lib/appmap/handler/rails/sql_handler.rb

```diff
@@ -43,6 +43,20 @@ class << self
             def examine(payload, sql:)
               return unless (examiner = build_examiner)
 
+              if AppMap.explain_queries? && examiner.in_transaction? && examiner.database_type == :postgres
```

Comment:
Why only do it in transaction?

Reply:
I want to use SAVEPOINT in case EXPLAIN something blows up. SAVEPOINT is illegal outside of a transaction.

Reply:
But it'd still be nice to EXPLAIN even outside of a transaction.